python_modules/dagster-test/dagster_test/test_project/build.sh

```diff
@@ -63,7 +63,7 @@ copy_py $ROOT/python_modules/dagster \
 
 echo -e "--- \033[32m:docker: Building Docker image\033[0m"
 
-PYTHON_SLIM_IMAGE="python:${PYTHON_VERSION}-slim"
+PYTHON_SLIM_IMAGE="python:${PYTHON_VERSION}"
```

Comment:
@johannkm do you think we should keep this as the slim version?

Reply:
let's make the change in another diff at least. If non-slim is required for local on m1, we'll have to consider if it would impact buildkite much to use the larger image, and if so make it overridable for local vs bk